docs/conf/australia/2018/sponsors/prospectus.rst

```diff
@@ -0,0 +1,202 @@
+:orphan:
+
+Write the Docs Day Australia 2018 Sponsorship
+##############################################
+
+.. raw:: pdf
+
+   PageBreak
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+Concept
+=======
+
+Write the Docs Day Australia (http://www.writethedocs.org/conf/australia/2018/) is a
+two day conference focusing on documentation systems, tech writing theory,
+and information delivery.
+
+Writing and maintaining documentation involves a multidisciplinary community
+of technical writers, designers, librarians, typesetters, developers,
+support teams, marketers and many others. This group of people can be
+collectively referred to as “documentarians”.
+
+Write the Docs creates a time and a place for this community to share
+information, discuss ideas, and work together to improve the art and
+science of documentation.
+
+For too long, people who care about documentation have felt alone in the
+world, not able to connect with their community. Write the Docs is a
+magical experience for many of our attendees, allowing them to feel
+like they have found their place. We believe this is the most important
+thing that our event can do, and we aim to provide that experience
+in Australia this year, and make it better than ever.
+
+Demographics
+============
+
+We hold two conferences yearly, one in Portland, USA in May, and one in
```

Comment:
We should update this to include Cincinnati and Australia instead of tagging Australia along on the end like that I think